### ISSUE TITLE
Could not update the project! Reason is: Command npm.cmd failed with exit code 1

### ISSUE THREAD

**Issue by devcodedao** [NativeScript/nativescript-cli#5440]
I updating nativescript latest version handmade but when run command ``tns update`` have errors appears 
Pls comment below if you have any solution 

![image](https://user-images.githubusercontent.com/44493646/98887738-1e481900-24c9-11eb-8ac4-f64d0d33a67b.png)
Thanks !

**Comment by sr3dna** [comment]
@rigor789 I was trying to update listview when I ran into that issue.  Thanks to you mentioning 'ns migrate'.   I was able to work around that 'command npm failed' issue by first migrating the listview project.   Here is what I did:

git reset --hard       (from root of nativescript-ui-samples-angular)
git clean -dfx
cd listview
ns migrate
npm install
ns update
(This time I don't get the 'Command npm failed with exit code 1' error.)

Hope this helps anyone troubleshooting.  Thanks
sr3dna
